js/interface.js

```diff
@@ -1352,6 +1352,19 @@ var DynamicLists = (function() {
         showAutocompleteOnFocus: true,
         createTokensOnBlur: false
       });
+      $('.sort-fields').html(tokenField({
```

Comment:
Similar code is already found about 26 lines above. This looks like unnecessary duplicates.

Reply:
@tonytlwu Yes, you are right. I'll remove it. The issue was that we were using an old variable at previous lines I've fixed that.